docs/reference/operations/index.md

```diff
@@ -0,0 +1,30 @@
+# Operations
```

Comment:
Could you add a _category_.json file and position this last (position 3) among the three files/folders in Reference? Right now it splits Accounts and Transfers in the docs site